script/Enums/HudEnums.lua

```diff
@@ -16,3 +16,9 @@ Enums.HudStyleNames = {
     "Balanced",
     "Tight",
 }
+
```

Comment:
can you add (language server)

`---@enum (key) enumName`

to all the enums here?

e.g.
```lua
---@enum (key) HudTacMapRanges
Enums.HudTacMapRanges = {
```

Reply:
It's difficult for me to accomplish the request you're making here. There are currently no Lua files in LTR in which an `---@enum` annotation has yet been added, nor can I find any examples of a Lua @enum annotation anywhere on the web, so there are no working examples for me to use. And the Lua language server documentation for "@enum" at https://github.com/LuaLS/lua-language-server/wiki/Annotations doesn't use `(key)`.

So I've done the best I can with the information I have, and updated the three enums in the file HudEnums.lua with annotations that follow the style given in the documentation cited above.

Reply:
Thanks for the link -- bookmarked.